package.json

```diff
@@ -5,7 +5,6 @@
   "type": "module",
   "main": "src/index.js",
   "module": "src/index.js",
-  "types": "types",
```

Comment:
🤔 Only question is for `types/test.ts`. How should users expect to consume that file?